INFO - 2021-11-05 10:14:40,466 - new_apply_order.py[line:882]: 开始第 starmoni【1】次进件……
INFO - 2021-11-05 10:14:40,467 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078480 ======
INFO - 2021-11-05 10:14:40,835 - new_apply_order.py[line:97]: 01636078480 检测用户存在成功！
INFO - 2021-11-05 10:14:40,891 - new_apply_order.py[line:106]: 01636078480 短信发送成功！
INFO - 2021-11-05 10:14:40,931 - new_apply_order.py[line:119]: 01636078480 注册成功！
INFO - 2021-11-05 10:14:40,976 - new_apply_order.py[line:135]: 01636078480 登录短信发送成功！
INFO - 2021-11-05 10:14:41,019 - new_apply_order.py[line:145]: userId=6227834270809595904,mobile=01636078480,token=4c3691cd122e4aa9ddfb8c6ca4419c40 登录成功
INFO - 2021-11-05 10:14:41,104 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:14:41,162 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:14:41,391 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:14:41,414 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:14:41,431 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:14:41,695 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:14:41,744 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:14:42,735 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:14:43,051 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:14:43,259 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:14:44,300 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:14:44,653 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:14:44,701 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:14:45,461 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:14:45,812 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:14:46,109 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:14:46,124 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:14:46,137 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:14:47,495 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:14:47,505 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:14:47,511 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:14:47,818 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:14:48,835 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:14:48,836 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444828396032000', 'mobile': '01636078480', 'order_status': 32}
INFO - 2021-11-05 10:14:48,850 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:14:48,862 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:14:50,217 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:14:50,236 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:14:50,245 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:14:50,749 - new_apply_order.py[line:905]: 校验重试查询第2次
ERROR - 2021-11-05 10:14:51,768 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-11-05 10:14:51,768 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444828396032000', 'mobile': '01636078480', 'order_status': 35}
INFO - 2021-11-05 10:14:52,280 - new_apply_order.py[line:905]: 校验重试查询第3次
INFO - 2021-11-05 10:14:53,290 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:14:53,290 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444828396032000', 'mobile': '01636078480', 'order_status': 40}
INFO - 2021-11-05 10:14:53,291 - new_apply_order.py[line:922]: ===========完成第【1】次进件===========
INFO - 2021-11-05 10:14:53,291 - new_apply_order.py[line:882]: 开始第 starmoni【2】次进件……
INFO - 2021-11-05 10:14:53,292 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078493 ======
INFO - 2021-11-05 10:14:53,643 - new_apply_order.py[line:97]: 01636078493 检测用户存在成功！
INFO - 2021-11-05 10:14:53,681 - new_apply_order.py[line:106]: 01636078493 短信发送成功！
INFO - 2021-11-05 10:14:53,724 - new_apply_order.py[line:119]: 01636078493 注册成功！
INFO - 2021-11-05 10:14:53,776 - new_apply_order.py[line:135]: 01636078493 登录短信发送成功！
INFO - 2021-11-05 10:14:53,822 - new_apply_order.py[line:145]: userId=6227834324454744064,mobile=01636078493,token=52809f4a06da4089151366e63d2809c8 登录成功
INFO - 2021-11-05 10:14:53,905 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:14:53,958 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:14:54,162 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:14:54,181 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:14:54,200 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:14:54,493 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:14:54,529 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:14:55,585 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:14:55,913 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:14:56,127 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:14:57,163 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:14:57,519 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:14:57,560 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:14:58,326 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:14:58,675 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:14:58,975 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:14:58,991 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:14:59,003 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:00,358 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:00,367 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:00,372 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:00,677 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:15:01,695 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:15:01,696 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444882057957376', 'mobile': '01636078493', 'order_status': 32}
INFO - 2021-11-05 10:15:01,710 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:01,721 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:03,081 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:03,101 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:03,111 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:03,616 - new_apply_order.py[line:905]: 校验重试查询第2次
ERROR - 2021-11-05 10:15:04,634 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-11-05 10:15:04,635 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444882057957376', 'mobile': '01636078493', 'order_status': 35}
INFO - 2021-11-05 10:15:05,144 - new_apply_order.py[line:905]: 校验重试查询第3次
INFO - 2021-11-05 10:15:06,162 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:15:06,163 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444882057957376', 'mobile': '01636078493', 'order_status': 40}
INFO - 2021-11-05 10:15:06,163 - new_apply_order.py[line:922]: ===========完成第【2】次进件===========
INFO - 2021-11-05 10:15:06,163 - new_apply_order.py[line:882]: 开始第 starmoni【3】次进件……
INFO - 2021-11-05 10:15:06,165 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078506 ======
INFO - 2021-11-05 10:15:06,524 - new_apply_order.py[line:97]: 01636078506 检测用户存在成功！
INFO - 2021-11-05 10:15:06,563 - new_apply_order.py[line:106]: 01636078506 短信发送成功！
INFO - 2021-11-05 10:15:06,603 - new_apply_order.py[line:119]: 01636078506 注册成功！
INFO - 2021-11-05 10:15:06,653 - new_apply_order.py[line:135]: 01636078506 登录短信发送成功！
INFO - 2021-11-05 10:15:06,699 - new_apply_order.py[line:145]: userId=6227834378481573888,mobile=01636078506,token=be3de6244f40b28be2c5fbdce1c832a4 登录成功
INFO - 2021-11-05 10:15:06,790 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:15:06,849 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:15:07,052 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:15:07,072 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:15:07,093 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:15:07,392 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:15:07,432 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:15:08,500 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:15:08,818 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:15:09,013 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:15:10,062 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:15:10,413 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:10,448 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:15:11,208 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:15:11,559 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:11,853 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:15:11,868 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:11,879 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:13,232 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:13,248 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:13,256 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:13,565 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:15:14,583 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:15:14,583 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444936126730240', 'mobile': '01636078506', 'order_status': 32}
INFO - 2021-11-05 10:15:14,597 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:14,607 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:15,991 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:16,013 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:16,022 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:16,524 - new_apply_order.py[line:905]: 校验重试查询第2次
ERROR - 2021-11-05 10:15:17,539 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-11-05 10:15:17,540 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444936126730240', 'mobile': '01636078506', 'order_status': 35}
INFO - 2021-11-05 10:15:18,049 - new_apply_order.py[line:905]: 校验重试查询第3次
INFO - 2021-11-05 10:15:19,068 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:15:19,068 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444936126730240', 'mobile': '01636078506', 'order_status': 40}
INFO - 2021-11-05 10:15:19,069 - new_apply_order.py[line:922]: ===========完成第【3】次进件===========
INFO - 2021-11-05 10:15:19,069 - new_apply_order.py[line:882]: 开始第 starmoni【4】次进件……
INFO - 2021-11-05 10:15:19,070 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078519 ======
INFO - 2021-11-05 10:15:19,419 - new_apply_order.py[line:97]: 01636078519 检测用户存在成功！
INFO - 2021-11-05 10:15:19,486 - new_apply_order.py[line:106]: 01636078519 短信发送成功！
INFO - 2021-11-05 10:15:19,543 - new_apply_order.py[line:119]: 01636078519 注册成功！
INFO - 2021-11-05 10:15:19,593 - new_apply_order.py[line:135]: 01636078519 登录短信发送成功！
INFO - 2021-11-05 10:15:19,639 - new_apply_order.py[line:145]: userId=6227834432726507520,mobile=01636078519,token=b5cd339bd64e9b84445c66eacd69eb8c 登录成功
INFO - 2021-11-05 10:15:19,728 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:15:19,785 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:15:20,006 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:15:20,027 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:15:20,048 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:15:20,299 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:15:20,334 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:15:21,338 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:15:21,677 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:15:21,894 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:15:22,936 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:15:23,295 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:23,335 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:15:24,148 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:15:24,503 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:24,828 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:15:24,843 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:24,854 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:26,219 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:26,233 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:26,240 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:26,549 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:15:27,565 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:15:27,565 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444990380052480', 'mobile': '01636078519', 'order_status': 32}
INFO - 2021-11-05 10:15:27,579 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:27,592 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:28,945 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:28,964 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:28,973 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:29,476 - new_apply_order.py[line:905]: 校验重试查询第2次
ERROR - 2021-11-05 10:15:30,489 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-11-05 10:15:30,489 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444990380052480', 'mobile': '01636078519', 'order_status': 35}
INFO - 2021-11-05 10:15:30,997 - new_apply_order.py[line:905]: 校验重试查询第3次
INFO - 2021-11-05 10:15:32,016 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:15:32,017 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456444990380052480', 'mobile': '01636078519', 'order_status': 40}
INFO - 2021-11-05 10:15:32,017 - new_apply_order.py[line:922]: ===========完成第【4】次进件===========
INFO - 2021-11-05 10:15:32,018 - new_apply_order.py[line:882]: 开始第 starmoni【5】次进件……
INFO - 2021-11-05 10:15:32,020 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078532 ======
INFO - 2021-11-05 10:15:32,371 - new_apply_order.py[line:97]: 01636078532 检测用户存在成功！
INFO - 2021-11-05 10:15:32,414 - new_apply_order.py[line:106]: 01636078532 短信发送成功！
INFO - 2021-11-05 10:15:32,457 - new_apply_order.py[line:119]: 01636078532 注册成功！
INFO - 2021-11-05 10:15:32,508 - new_apply_order.py[line:135]: 01636078532 登录短信发送成功！
INFO - 2021-11-05 10:15:32,552 - new_apply_order.py[line:145]: userId=6227834486912720896,mobile=01636078532,token=c3afce7dfd8c07cd8eb5864311748327 登录成功
INFO - 2021-11-05 10:15:32,632 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:15:32,683 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:15:32,893 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:15:32,914 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:15:32,930 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:15:33,180 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:15:33,217 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:15:34,232 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:15:34,552 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:15:34,739 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:15:35,788 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:15:36,147 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:36,191 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:15:36,970 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:15:37,323 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:37,631 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:15:37,646 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:37,657 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:39,017 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:39,028 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:39,034 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:39,342 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:15:40,352 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:15:40,352 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445044490768384', 'mobile': '01636078532', 'order_status': 32}
INFO - 2021-11-05 10:15:40,360 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:40,367 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:41,719 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:41,738 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:41,747 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:42,249 - new_apply_order.py[line:905]: 校验重试查询第2次
INFO - 2021-11-05 10:15:43,264 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:15:43,264 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445044490768384', 'mobile': '01636078532', 'order_status': 40}
INFO - 2021-11-05 10:15:43,265 - new_apply_order.py[line:922]: ===========完成第【5】次进件===========
INFO - 2021-11-05 10:15:43,265 - new_apply_order.py[line:882]: 开始第 starmoni【6】次进件……
INFO - 2021-11-05 10:15:43,266 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078543 ======
INFO - 2021-11-05 10:15:43,614 - new_apply_order.py[line:97]: 01636078543 检测用户存在成功！
INFO - 2021-11-05 10:15:43,654 - new_apply_order.py[line:106]: 01636078543 短信发送成功！
INFO - 2021-11-05 10:15:43,694 - new_apply_order.py[line:119]: 01636078543 注册成功！
INFO - 2021-11-05 10:15:43,747 - new_apply_order.py[line:135]: 01636078543 登录短信发送成功！
INFO - 2021-11-05 10:15:43,795 - new_apply_order.py[line:145]: userId=6227834534056697856,mobile=01636078543,token=0052e5b366f63abdbcf081d500f0f8f2 登录成功
INFO - 2021-11-05 10:15:43,886 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:15:43,939 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:15:44,129 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:15:44,146 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:15:44,164 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:15:44,418 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:15:44,460 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:15:45,493 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:15:45,801 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:15:46,003 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:15:47,043 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:15:47,400 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:47,443 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:15:48,208 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:15:48,559 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:48,860 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:15:48,875 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:48,886 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:15:50,246 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:50,260 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:15:50,266 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:15:50,574 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:15:51,591 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-11-05 10:15:51,592 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445091680882688', 'mobile': '01636078543', 'order_status': 35}
INFO - 2021-11-05 10:15:52,102 - new_apply_order.py[line:905]: 校验重试查询第2次
INFO - 2021-11-05 10:15:53,118 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:15:53,118 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445091680882688', 'mobile': '01636078543', 'order_status': 40}
INFO - 2021-11-05 10:15:53,119 - new_apply_order.py[line:922]: ===========完成第【6】次进件===========
INFO - 2021-11-05 10:15:53,119 - new_apply_order.py[line:882]: 开始第 starmoni【7】次进件……
INFO - 2021-11-05 10:15:53,121 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078553 ======
INFO - 2021-11-05 10:15:53,470 - new_apply_order.py[line:97]: 01636078553 检测用户存在成功！
INFO - 2021-11-05 10:15:53,513 - new_apply_order.py[line:106]: 01636078553 短信发送成功！
INFO - 2021-11-05 10:15:53,555 - new_apply_order.py[line:119]: 01636078553 注册成功！
INFO - 2021-11-05 10:15:53,605 - new_apply_order.py[line:135]: 01636078553 登录短信发送成功！
INFO - 2021-11-05 10:15:53,652 - new_apply_order.py[line:145]: userId=6227834575404146688,mobile=01636078553,token=4c62cee770262fcc290c835837286e4d 登录成功
INFO - 2021-11-05 10:15:53,738 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:15:53,790 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:15:53,985 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:15:54,002 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:15:54,018 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:15:54,281 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:15:54,319 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:15:55,327 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:15:55,626 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:15:55,820 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:15:56,865 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:15:57,222 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:57,264 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:15:58,054 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:15:58,420 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:15:58,736 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:15:58,750 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:15:58,760 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:16:00,112 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:00,126 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:16:00,134 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:16:00,443 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:16:01,461 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:16:01,461 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445133011554304', 'mobile': '01636078553', 'order_status': 32}
INFO - 2021-11-05 10:16:01,475 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:16:01,485 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:16:02,840 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:02,858 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:16:02,868 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:16:03,372 - new_apply_order.py[line:905]: 校验重试查询第2次
INFO - 2021-11-05 10:16:04,391 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:16:04,391 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445133011554304', 'mobile': '01636078553', 'order_status': 40}
INFO - 2021-11-05 10:16:04,392 - new_apply_order.py[line:922]: ===========完成第【7】次进件===========
INFO - 2021-11-05 10:16:04,392 - new_apply_order.py[line:882]: 开始第 starmoni【8】次进件……
INFO - 2021-11-05 10:16:04,394 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078564 ======
INFO - 2021-11-05 10:16:04,747 - new_apply_order.py[line:97]: 01636078564 检测用户存在成功！
INFO - 2021-11-05 10:16:04,801 - new_apply_order.py[line:106]: 01636078564 短信发送成功！
INFO - 2021-11-05 10:16:04,856 - new_apply_order.py[line:119]: 01636078564 注册成功！
INFO - 2021-11-05 10:16:04,931 - new_apply_order.py[line:135]: 01636078564 登录短信发送成功！
INFO - 2021-11-05 10:16:04,990 - new_apply_order.py[line:145]: userId=6227834622783004672,mobile=01636078564,token=f3b4817192261d16fd9883820fd6dac1 登录成功
INFO - 2021-11-05 10:16:05,090 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:16:05,153 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:16:05,354 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:16:05,367 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:16:05,380 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:16:05,662 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:16:05,702 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:16:06,715 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:16:07,033 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:16:07,226 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:16:08,272 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:16:08,627 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:08,666 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:16:09,471 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:16:09,823 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:10,125 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:16:10,139 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:16:10,155 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:16:11,561 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:11,577 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:16:11,585 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:16:11,891 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:16:12,906 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:16:12,907 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445180654653440', 'mobile': '01636078564', 'order_status': 32}
INFO - 2021-11-05 10:16:12,921 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:16:12,931 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:16:14,297 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:14,315 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:16:14,324 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:16:14,830 - new_apply_order.py[line:905]: 校验重试查询第2次
INFO - 2021-11-05 10:16:15,849 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:16:15,849 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445180654653440', 'mobile': '01636078564', 'order_status': 40}
INFO - 2021-11-05 10:16:15,849 - new_apply_order.py[line:922]: ===========完成第【8】次进件===========
INFO - 2021-11-05 10:16:15,850 - new_apply_order.py[line:882]: 开始第 starmoni【9】次进件……
INFO - 2021-11-05 10:16:15,851 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078575 ======
INFO - 2021-11-05 10:16:16,199 - new_apply_order.py[line:97]: 01636078575 检测用户存在成功！
INFO - 2021-11-05 10:16:16,272 - new_apply_order.py[line:106]: 01636078575 短信发送成功！
INFO - 2021-11-05 10:16:16,376 - new_apply_order.py[line:119]: 01636078575 注册成功！
INFO - 2021-11-05 10:16:16,447 - new_apply_order.py[line:135]: 01636078575 登录短信发送成功！
INFO - 2021-11-05 10:16:16,498 - new_apply_order.py[line:145]: userId=6227834671017500672,mobile=01636078575,token=a9761652f88086c4360d7edff27a1625 登录成功
INFO - 2021-11-05 10:16:16,590 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:16:16,692 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:16:16,952 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:16:16,974 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:16:16,993 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:16:17,281 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:16:17,326 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:16:18,262 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:16:18,592 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:16:18,780 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:16:19,825 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:16:20,179 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:20,226 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:16:21,023 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:16:21,372 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:21,614 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:16:21,629 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:16:21,640 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:16:22,997 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:23,010 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:16:23,017 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:16:23,325 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:16:24,342 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:16:24,343 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445229065310208', 'mobile': '01636078575', 'order_status': 32}
INFO - 2021-11-05 10:16:24,356 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:16:24,367 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:16:25,718 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:25,735 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:16:25,744 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:16:26,245 - new_apply_order.py[line:905]: 校验重试查询第2次
INFO - 2021-11-05 10:16:27,260 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:16:27,260 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445229065310208', 'mobile': '01636078575', 'order_status': 40}
INFO - 2021-11-05 10:16:27,261 - new_apply_order.py[line:922]: ===========完成第【9】次进件===========
INFO - 2021-11-05 10:16:27,261 - new_apply_order.py[line:882]: 开始第 starmoni【10】次进件……
INFO - 2021-11-05 10:16:27,262 - new_apply_order.py[line:84]: ======【nga测试环境】生成手机号：01636078587 ======
INFO - 2021-11-05 10:16:27,609 - new_apply_order.py[line:97]: 01636078587 检测用户存在成功！
INFO - 2021-11-05 10:16:27,650 - new_apply_order.py[line:106]: 01636078587 短信发送成功！
INFO - 2021-11-05 10:16:27,695 - new_apply_order.py[line:119]: 01636078587 注册成功！
INFO - 2021-11-05 10:16:27,745 - new_apply_order.py[line:135]: 01636078587 登录短信发送成功！
INFO - 2021-11-05 10:16:27,793 - new_apply_order.py[line:145]: userId=6227834718580908032,mobile=01636078587,token=716423ea347394608702775c6f80c17e 登录成功
INFO - 2021-11-05 10:16:27,882 - new_apply_order.py[line:212]: 个人信息提交成功！
INFO - 2021-11-05 10:16:27,934 - new_apply_order.py[line:223]: 联系人信息提交成功！
INFO - 2021-11-05 10:16:28,139 - new_apply_order.py[line:255]: 创建预审订单成功！
INFO - 2021-11-05 10:16:28,159 - new_apply_order.py[line:263]: 预审状态修改为通过,更新条数 1
INFO - 2021-11-05 10:16:28,176 - new_apply_order.py[line:266]: 更新初始额度！更新条数 1
INFO - 2021-11-05 10:16:28,429 - new_apply_order.py[line:342]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-11-05 10:16:28,470 - new_apply_order.py[line:352]: 视频认证通过！
INFO - 2021-11-05 10:16:29,449 - new_apply_order.py[line:510]: bvn和银行卡绑卡认证通过！
INFO - 2021-11-05 10:16:29,771 - new_apply_order.py[line:561]: user更新1条
INFO - 2021-11-05 10:16:29,966 - new_apply_order.py[line:612]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-11-05 10:16:31,010 - new_apply_order.py[line:618]: 订单表更新1条，信审更新1条
INFO - 2021-11-05 10:16:31,365 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:31,412 - new_apply_order.py[line:647]: 订单回调复审通过成功！
INFO - 2021-11-05 10:16:32,215 - new_apply_order.py[line:669]: 获取token成功！token=bearer d5992460-d0b8-41c1-a203-8106a462060d！
INFO - 2021-11-05 10:16:32,567 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:32,882 - new_apply_order.py[line:687]: 后台已触发放款成功！
INFO - 2021-11-05 10:16:32,897 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:16:32,909 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:16:34,268 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:34,283 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:16:34,291 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:16:34,599 - new_apply_order.py[line:905]: 校验重试查询第1次
ERROR - 2021-11-05 10:16:35,621 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是32
INFO - 2021-11-05 10:16:35,622 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445276209287168', 'mobile': '01636078587', 'order_status': 32}
INFO - 2021-11-05 10:16:35,639 - new_apply_order.py[line:701]: xxl_job web 登录成功！
INFO - 2021-11-05 10:16:35,649 - new_apply_order.py[line:717]: xxl_job 放款准备作业执行成功！
INFO - 2021-11-05 10:16:37,007 - new_apply_order.py[line:632]: 查询loan_order_id成功！
INFO - 2021-11-05 10:16:37,021 - new_apply_order.py[line:725]: 修改放款准备完成时间成功！
INFO - 2021-11-05 10:16:37,028 - new_apply_order.py[line:733]: xxl_job 放款完成作业执行成功！
INFO - 2021-11-05 10:16:37,534 - new_apply_order.py[line:905]: 校验重试查询第2次
ERROR - 2021-11-05 10:16:38,551 - new_apply_order.py[line:801]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-11-05 10:16:38,552 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445276209287168', 'mobile': '01636078587', 'order_status': 35}
INFO - 2021-11-05 10:16:39,063 - new_apply_order.py[line:905]: 校验重试查询第3次
INFO - 2021-11-05 10:16:40,079 - new_apply_order.py[line:799]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-11-05 10:16:40,079 - new_apply_order.py[line:802]: {'app': 'starmoni', 'loan_order_id': '1456445276209287168', 'mobile': '01636078587', 'order_status': 40}
INFO - 2021-11-05 10:16:40,080 - new_apply_order.py[line:922]: ===========完成第【10】次进件===========
INFO - 2021-11-05 10:16:40,080 - new_apply_order.py[line:925]: ======完成第【10】次进件,耗时119.61s=====
